src/api/v1/utils.ts

```diff
@@ -5,6 +5,28 @@ import * as Hapi from 'hapi';
 import * as Boom from 'boom';
 import { ApiResponse } from './schema/response';
 
+export interface JoiBoomError extends Boom {
+  details: {message: string}[];
+}
+
+/*
+ * formatJoiValidation allows joi errors thrown from
+ * failAction to be reformatted before sending response.
+ * Note joi throws an error on the first value that fails
```

Comment:
This is configurable. Do we want to include all fields that fail validation?

Reply:
This may be useful for signup routes where we are taking a bigger number of fields from the user, but for now I think we should leave it as it is.